src/pretix/presale/templates/pretixpresale/event/checkout_addons.html

```diff
@@ -16,7 +16,7 @@
             {% trans "A product in your cart is only sold in combination with add-on products that are no longer available. Please contact the event organizer." %}
         </div>
     {% endif %}
-    <form class="form-horizontal" method="post" data-asynctask
+    <div class="form-horizontal" method="post" data-asynctask
```

Comment:
Why did you remove the form? It does not work without it in my tests.

```suggestion
    <form class="form-horizontal" method="post" data-asynctask
```